Formula/lazygit.rb

```diff
@@ -23,12 +23,12 @@ def install
     system "go", "build", "-mod=vendor", *std_go_args(ldflags: ldflags)
   end
 
-  # lazygit is a terminal GUI, but it can be run in 'client mode' for example to write to git's todo file
+  # lazygit is a terminal GUI, but it can be run in 'client mode' to do certain tasks
   test do
-    (testpath/"git-rebase-todo").write ""
-    ENV["LAZYGIT_DAEMON_KIND"] = "INTERACTIVE_REBASE"
-    ENV["LAZYGIT_REBASE_TODO"] = "foo"
-    system "#{bin}/lazygit", "git-rebase-todo"
-    assert_match "foo", (testpath/"git-rebase-todo").read
+    ENV["LAZYGIT_DAEMON_KIND"] = "1" # exit immediately
+    ENV["LAZYGIT_DAEMON_INSTRUCTION"] = "{}"
+    ok = system "#{bin}/lazygit"
+    # assert that the command exits with exit code 0
+    assert_equal true, ok
```

Comment:
System already fails if it does

Reply:
I'm surprised if that's true because the test fails upon comparing true to nil. If system raised an exception upon failure, I wouldn't expect it to reach the next line.